mksnapshot.sh

```diff
@@ -10,40 +10,34 @@
 echo "RUN: $0"
 if [ $# -lt 1 ]; then
 	echo "Usage: $0 [branch]"
-	echo "Where [branch] is the path under /bzr/ to the branch to snapshot."
+	echo "Where [branch] is the name of the branch to snapshot."
```

Comment:
I would replace `branch` with `revision` (or similar) because one does not need a git branch to create a snapshot:

```
Where [revision] is the git tree-ish to snapshot (e.g., a branch name)
```

Creating a snapshot of a tagged version or a commit is a perfectly reasonable thing to do. This minor change does not affect the script code much.

N.B. Tree-ish is documented in "man git". Your `git archive` expects a tree-ish argument.